code/modules/mob/living/carbon/human/human.dm

```diff
@@ -2016,4 +2017,13 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 	var/obj/item/organ/internal/lantern/O = get_int_organ(/obj/item/organ/internal/lantern)
 	if(O && O.glowing)
 		O.toggle_biolum(TRUE)
-		visible_message("<span class='danger'>[src] is engulfed in shadows and fades into the darkness.</span>", "<span class='danger'>A sense of dread washes over you as you suddenly dim dark.</span>")
\ No newline at end of file
+		visible_message("<span class='danger'>[src] is engulfed in shadows and fades into the darkness.</span>", "<span class='danger'>A sense of dread washes over you as you suddenly dim dark.</span>")
+
+/mob/living/carbon/human/can_use_machinery()
+	if(!IsAdvancedToolUser())

```

Comment:
This check should be added to the mob-level proc definition instead, as IsAdvancedToolUser() is actually defined on the mob level as well.

Looks good otherwise!